src/pages/index.js

```diff
@@ -32,7 +32,7 @@ const features = [
   {
     title: "APIs & Tools",
     imageUrl: "img/apisclients.png",
-    url: "/docs/apis-tools/working-with-apis-tools/",
+    url: "/",
```

Comment:
What's going on here and in L44? This would bork the clickable icon into these sections. Is this because these didn't technically exist yet?

Reply:
Yes, exactly. Those pages didn't exist in this version. Removing the blocks makes the UI look pretty bad, so I chose to leave them in place, but point them at basically nothing.

Reply:
I'm good with this!